lib/ReactViews/BottomDock/Timeline/DateTimePicker.tsx

```diff
@@ -250,6 +275,15 @@ class DateTimePicker extends React.Component<PropsType> {
     window.removeEventListener("click", this.closePickerEventHandler);
   }
 
+  componentDidUpdate() {
+    // Scroll into view to reveal the selected time item
```

Comment:
this will fire whenever any of the props for the component change. It might be better to abstract out the <Grid className="time-grid"> JSX into its own component and then use an effect to conditionally scroll the time into view. This was a very nice touch though :)

Reply:
Good point! Extracting the JSX would be a bigger change I would hold off until a grand rewrite of the component. For now I have added a check with prevProps for `currentDate`.